engine/apps/public_api/views/incidents.py

```diff
@@ -112,3 +114,74 @@ def destroy(self, request, *args, **kwargs):
             wipe.apply_async((instance.pk, request.user.pk))
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+    @action(methods=["post"], detail=True)
```

Comment:
These API actions mostly duplicate the [internal API logic](https://github.com/grafana/oncall/blob/971384b50e2e4324fecd480a8abaa4cb8b7d78be/engine/apps/api/views/alert_group.py#L503-L554). I think these alert group state changes should be eventually refactored to be a proper FSM that can be reused in both APIs, but I want to leave this outside of the scope of this PR/issue.

Reply:
+1